daml-service/entrypoint.sh

```diff
@@ -3,7 +3,7 @@ set -euo pipefail
 
 [[ -n ${VERBOSE:-} ]] && set -x
 
-# Log everything (stdout and stderr) to both log file and terminal
-exec > >(tee /var/log/promtail/"${DAML_SERVICE}".log) 2>&1
+# Bash trick - Log everything (stdout and stderr) to both log file and terminal
+exec > >(tee -a "/var/log/promtail/${DAML_SERVICE}.log") 2>&1
 
-eval "exec java -XX:+CrashOnOutOfMemoryError ${JAVA_OPTS:-} -jar /opt/service.jar \"\$@\""
+eval "exec java -jar /opt/${DAML_SERVICE}.jar \"\$@\""
```

Comment:
Why remove the `CrashOnOutOfMemoryError` though? How is that related to switching to `JDK_JAVA_OPTIONS`?

Reply:
I don't think we need that for the demo.  I removed it because it may have been a premature kill for the Canton container.  I did leave a comment that it can be added by the user.

Reply:
Moved `-XX:+CrashOnOutOfMemoryError` to `JDK_JAVA_OPTIONS` in docker compose files so its explicit to users